src/Integration/Binding/BindingProcessImpl.cs

```diff
@@ -127,6 +134,11 @@ public async Task<bool> DownloadQualityProfileAsync(IProgress<FixedStepsProgress
             {
                 var serverLanguage = language.ServerLanguage;
 
+                if (serverLanguage == null)
```

Comment:
For some reason, ProjectToLanguageMapper returns a list with `Language.Unknown` if no language was detected. I don't know why this is the behavior, and I'm not sure what will break if I change it. So I'm safe-guarding for it here.